system/lib/pthread/library_pthread.c

```diff
@@ -31,6 +31,8 @@ char *gets(char *);
 // Extra pthread_attr_t field:
 #define _a_transferredcanvases __u.__s[9]
 
+void __pthread_testcancel();
```

Comment:
Because library_pthread.c has the emscripten-specific implementation of various functions provided by musl, for example pthread_testcancel. Musl changed its implementation to implement functions prefixed, and then alias them unprefixed.
I don't fully remember the subtleties here, but I believe it has to do with symbol resolution of functions vs aliases at link time. This mirrors musl's expected implementation, and we re-alias it to `pthread_testcancel` at the end of library_pthread.c.
For user-facing code the interface is the same (unprefixed).